packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -1094,6 +1094,7 @@ export const prepareVaultManagerKit = (
           state.lockedQuote = undefined;
 
           if (vaultData.getSize() === 0) {
+            void helper.writeMetrics();
```

Comment:
on line 1160 it awaits before returning. Which is correct?

also that one superfluously references `facets`.

Reply:
> on line 1160 it awaits before returning. Which is correct?

There were more calls that said `void` than `await`, so ...   No, that's not a good justification.

I don't see any reason that the calling environment wants to respond to the result or wait for it to finish, so I changed them all to `void`.

> also that one superfluously references facets.
fixed